packages/amplify_flutter/example/pubspec.yaml

```diff
@@ -21,9 +21,18 @@ dependencies:
     sdk: flutter
 
 dev_dependencies:
+  amplify_analytics_pinpoint: ^0.2.4
```

Comment:
Should these versions be changed to match RC version?

Reply:
I think relative paths would be consistent with the other example projects. Not sure if there is a reason for this to be different?

Reply:
Fixed